rts/Game/GlobalUnsynced.h

```diff
@@ -48,8 +48,10 @@ class CGlobalUnsynced {
 	 * i.e. it means that 5% of the total
 	 * cpu time is spend for drawing and 95%
 	 * for reconnecting/simulation.
+	 * Note: this isn't necessary because the
+	 * engine enforces minDrawFPS.
 	 */
-	static constexpr float reconnectSimDrawBalance = 0.05f;
+	static constexpr float reconnectSimDrawBalance = 0.00f;
```

Comment:
Is this meant to be `0.00f`?

Reply:
That's why I added the comment:
/* Note: this isn't necessary because the
	 * engine enforces minDrawFPS.

I don't believe we need it above 0.

Reply:
The comment's wording still mentions 5% and 95% though, perhaps remove that part or reword it, for example `if the constant were set to 0.05f, ...`